README.md

```diff
@@ -21,14 +21,24 @@ Below are a few key features of the Performance Dashboard:
 
 To access additional details on the system's features and workflows, please view the [user guide](docs/user-guide.pdf).
 
-## Deployment
+## Getting Started
+
+### Launch Solution
 
 PDoA comes with pre-built code to provision an instance in your AWS account. You can use the provided AWS CloudFormation [template](docs/installation.md#-deploying-with-aws-cloudformation-template) (CFT) to install PDoA in a few clicks. If you're in a developer or techops role and want to update or experiment with the source code, the provided AWS Cloud Development Kit (CDK) [script](docs/installation.md#deploying-with-aws-cloud-development-kit-cdk) makes it easy to deploy your changes. If you're already familiar with deploying resources on AWS using CFT, click the link below to deploy in the us-east-1 region. Refer to the [installation guide](docs/installation.md) for more detailed instructions.
 
 | Region               | Launch                                                                                                                                                                                                                                                            |
 | -------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | Install in us-east-1 | [![Install in us-east-1](docs/images/launch-stack.svg)](https://console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/quickcreate?templateURL=https://performance-dashboard-on-aws-us-east-1.s3.amazonaws.com/performance-dashboard-us-east-1.json) |
 
+### Clone this repository
+
+The mainline branch of this repository develops rapidly. If you want to obtain the latest stable copy of the code, clone this repository using the tag of the most recent release, which currently is 0.2.0-beta.
+
+```
+$ git clone -b 0.2.0-beta https://github.com/awslabs/performance-dashboard-on-aws.git
+```
+
```

Comment:
Shouldn't this be on the installation instructions instead?

Reply:
Users may miss it if we put it in the Installation doc